pkg/zfs/zfs_util.go

```diff
@@ -292,13 +296,56 @@ func buildVolumeResizeArgs(vol *apis.ZFSVolume) []string {
 	return ZFSVolArg
 }
 
+// builldVolumeBackupArgs returns volume send command for sending the zfs volume
+func buildVolumeBackupArgs(bkp *apis.ZFSBackup, vol *apis.ZFSVolume) []string {
+	var ZFSVolArg []string
+	backupDest := bkp.Spec.BackupDest
+
+	bkpAddr := strings.Split(backupDest, ":")
```

Comment:
q_: how does this work with destination configured with credentials?

Reply:
credentials or managed by velero. Here it is plain tcp connect to te velero plugin. Then velero plugin reads this stream and uploca/download it to the cloud.